### ISSUE TITLE
Property for Marker color

### ISSUE THREAD

**Issue by NickAusloos** [nativescript-community/ui-mapbox#18]
Is there something on the roadmap to set a Color property for the Marker?

**Comment by cvietor** [comment]
No, the `addMarker` method uses mapbox annotation types (deprecated in mapbox android sdk) under the hood and they don't support a color property, as far as i can see.

- [android sdk docs MarkerOptions](https://docs.mapbox.com/archive/android/maps/api/8.6.6/index.html)
- [ios sdk docs MGLAnnotation](https://docs.mapbox.com/ios/maps/api/6.3.0/Protocols/MGLAnnotation.html)

you can consider to use SymbolLayers, [have a look at the samples](https://github.com/nativescript-community/ui-mapbox/blob/master/demo/app/main-view-model.ts#L615-L670)
